src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -100,6 +151,35 @@ internal static string GetAccountId(string functionArn)
             return null;
         }
 
+        private static string GetFaasId(string functionArn)
+        {
+            var faasId = functionArn;
+
+            // According to faas.id description https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-lambda.md#all-triggers
+            // the 8th part of arn (version, see https://docs.aws.amazon.com/lambda/latest/dg/configuration-versions.html#versioning-versions-using)
+            // should not be included into faas.id
+            var items = functionArn.Split(':');
+            if (items.Length >= 8)
+            {
+                var result = new string[7];
+                Array.Copy(items, result, 7);
+                faasId = string.Join(":", result);
```

Comment:
This looks like C-code... Isn't there a cool C# 8slicing feature we could use to have less code and maybe even elide the copy? Would `string.Join(":", items[..7])` work?

Reply:
It requires at least NET Standard 2.1, but it still can be simplified with LINQ expression: `string.Join(":", items.Take(7));`.

Reply:
Doesn't OTel take a dependency a backport of the Span features anyway?